cores/cv32e40p/user_manual/source/instruction_set_extensions.rst

```diff
@@ -317,6 +317,16 @@ together.
 -  Immediate branching instructions are useful to compare a register
    with an immediate value before taking or not a branch, see see :ref:`pulp_immediate_branching`.
 
+Extract, Insert, Clear and Set instructions have the following meaning:
+
+- Extract Is3+1/rs2[9:5]+1 bits from position Is2/rs2[4:0] [and sign extend it]
```

Comment:
Can you use () brackets to make the expression more clear? Same remark for below expressions

Reply:
done